expat/CMakeLists.txt

```diff
@@ -302,6 +302,13 @@ if(NOT WIN32)
     set_property(TARGET ${_EXPAT_TARGET} PROPERTY NO_SONAME ${NO_SONAME})
 endif(NOT WIN32)
 
+target_include_directories(${_EXPAT_TARGET}
+    INTERFACE
+        $<BUILD_INTERFACE:${CMAKE_CURRENT_SOURCE_DIR}/lib>
+        $<BUILD_INTERFACE:${CMAKE_CURRENT_BINARY_DIR}>
+        $<INSTALL_INTERFACE:${CMAKE_INSTALL_INCLUDEDIR}>
+)
+
```

Comment:
I can confirm that this addition — with or without the `INSTALL_INTERFACE` line — fixes #380 for me locally. Could you either limit this pull request to a commit with just this fix OR make a new pull request containing of just that fix?

If we still need a fix for `-- Old export file "..." will be replaced.  Removing files [...]."` — new files seem to be installed right after for me but I might be missing something — it allows us to talk about that in isolation, after. Is that an okay way moving forward?

Reply:
INSTALL_INTERFACE line is for "install_dir/lib/cmake/expat-2.2.9/expat.cmake" not for add_subdirectory. It is needed in find_package CONFIG mode.

Old export file "..." will be replaced. Removing files [...]. - is only when We change some in exported target like target_include_directories and it is not very important. Normal users will never see it. ;)